### DISCUSSION TITLE
Any plans for spatial query support - PostGIS?

### DISCUSSION THREAD

**Discussion by bluespider42** [geldata/gel#3692] [Q&A] [answered]
Are there any plans for enabling extensions to the query language to support spatial queries on a PostGIS enabled postgres instance?

**Comment by tailhook** [comment, marked as answer]
Yes. It's on the roadmap, but no specific ETA yet.

**Reply by trompx** [reply]
Is it the only extension on the roadmap or all extensions will be supported (like ltree)? Is there a way in the meantime to use edgeDB for some stuff and an ORM/query builder for non supported fieds/extensions (how would that work as we define the schema with edgeDB)?
